docs/documentation/GettingStarted.md

```diff
@@ -53,9 +53,9 @@ The following parameters can be configured in the *psdk_wrapper/cfg/psdk_params.
 | mandatory_modules             |           |                                    |                                             |
 | - telemetry                   | Bool      |  True                              | Trigger node failure, if module not loaded  |
 | - flight_control              | Bool      |  True                              | Trigger node failure, if module not loaded  |
-| - camera                      | Bool      |  True                              | Trigger node failure, if module not loaded  |
-| - gimbal                      | Bool      |  True                              | Trigger node failure, if module not loaded  |
-| - liveview                    | Bool      |  True                              | Trigger node failure, if module not loaded  |
+| - camera                      | Bool      |  False                             | Trigger node failure, if module not loaded  |
```

Comment:
add hms module as a mandatory one

Reply:
The module is not considered optional in the development so far, but I can see how that would be desirable. And if that is the case, it is probably the proper time to add this. What do you think?

Reply:
Well we follow the convention that all the modules should support being mandatory or not. Just like `telemetry`. The only thing I'm saying is to add the default value for it to be True (meaning Mandatory)

Reply:
Got it, it's been implemented 🦾